### PR TITLE
Replace hardcoded base urls

### PR DESCRIPTION
**Pull request by dandehoon**
#### :heavy_check_mark: Checklist

- [ ] All tests are passing `yarn test`
- [ ] Screenshots attached (for UI changes)
- [ ] Relevant documentation updated
- [ ] Prettier run on changed files
- [ ] Tests added for new functionality
- [ ] Regression tests added for bug fixes

### CODE REVIEW

File: plugins/auth-backend/src/service/router.ts [hunk 1]
```diff
@@ -56,29 +59,29 @@ export async function createRouter(
   // TODO: read from app config
```
Comment (freben) [inline plugins/auth-backend/src/service/router.ts:59]: remove todo? :)

**Review by freben — APPROVED** [review]

File: plugins/auth-backend/src/service/router.ts [hunk 2]
```diff
@@ -36,14 +36,17 @@ export async function createRouter(
   const router = Router();
   const logger = options.logger.child({ plugin: 'auth' });
 
-  const baseUrl = `${options.config.getString('backend.baseUrl')}/auth`;
+  const appUrl = options.config.getString('app.baseUrl');
```
Comment (Rugvip) [inline plugins/auth-backend/src/service/router.ts:39]: Should make this `appOrigin`, i.e. parsing the url to extract the origin.

**Review by Rugvip — COMMENTED** [review]
Thanks! Needs a small fix and then good to go by me

File: plugins/auth-backend/src/service/router.ts (hunk 2 above)
Comment (danztran) [reply plugins/auth-backend/src/service/router.ts]: Thanks, updated!

Comment (freben) [reply plugins/auth-backend/src/service/router.ts]: @Rugvip was this the way you intended? did you mean to actually URL parse `app.baseUrl` and then extract only host and port?

Comment (Rugvip) [reply plugins/auth-backend/src/service/router.ts]: Yeah we need to actually parse it and extract the origin, i.e. `new URL('http://localhost:3000/my/path').origin`

File: plugins/auth-backend/src/service/router.ts [hunk 3]
```diff
@@ -36,14 +36,17 @@ export async function createRouter(
   const router = Router();
   const logger = options.logger.child({ plugin: 'auth' });
 
-  const baseUrl = `${options.config.getString('backend.baseUrl')}/auth`;
+  const appOrigin = options.config.getString('app.baseUrl');
```
Comment (Rugvip) [inline plugins/auth-backend/src/service/router.ts:39]: Not sure if it was clear in the thread so wanna reiterate that the `app.baseUrl` needs to be parsed to extract only the origin. The baseUrl may contain path or even query parts that we don't want included in the origin.

**Review by Rugvip — CHANGES_REQUESTED** [review]

**Review by Rugvip — APPROVED** [review]
Looks good, thanks! 🎉

**Pull request merged**